lib/environment/EnvironmentsController.js

```diff
@@ -237,6 +240,47 @@ class EnvironmentsController extends BaseController {
       done(null, cmdResult);
     });
   }
+
+  export(options, done) {
+    let env;
+
+    const cmdResult = new CommandResult();
+
+    async.waterfall([
+      (next) => {
+        this.environmentsService.getActiveOrSpecified(options, (err, data) => {
+          if (err) {
+            return next(err);
+          }
+          env = data;
+
+          console.log(JSON.stringify(env, null, 4));
+
+          const envFileExporter = new EnvFileExporter_1_0_0(this.options);
+          const exportOptions = { env: env };
+          envFileExporter.export(exportOptions, next);
+        });
+      },
+      (exportedConfig, next) => {
+        //console.log(JSON.stringify(exportedConfig, null, 4));
+
+        const filename = options.file;
+        fs.writeFileSync(filename, JSON.stringify(exportedConfig, null, 4));
```

Comment:
There's a helper method for this in `Utils.js` - `Utils.writeJSON`
...Also, given that this is the sync version of file writing what will happen on error? Will it throw?